test/fixtures/exports-field/node_modules/exports-field/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "exports-field",
+  "version": "1.0.0",
+  "main": "./main.js",
+  "browser": {
+    "./lib/main.js": "./lib/browser.js"
+  },
+  "exports": {
+    ".": "./index.js",
+    "./lib/": {
```

```suggestion
    "./inner/": {
```

otherwise most test cases would also succeed if it doesn't work